src/buttonMaker/IconForm.jsx

```diff
@@ -0,0 +1,98 @@
+import * as allIcons from "@chakra-ui/icons";
+import "./buttonMaker.css";
+import { changeForm, clickForm } from "./formLogic";
+
+// A module of all the availabe icons
+let { Icon, createIcon, ...iconsModule } = allIcons;
+
+// A list of all the icon names
+let iconsName = Object.keys(iconsModule).map((e) => e);
```

Comment:
issue: this `map` doesn't do anything :stuck_out_tongue:

Reply:
hehehe... my bad